flexget/plugins/output/decompress.py

```diff
@@ -54,7 +54,7 @@ def get_output_path(to, entry):
         else:
             return os.path.dirname(entry.get('location'))
     except RenderError:
-        raise PluginError('Could not render path: %s', to)
+        raise plugin.PluginError('Could not render path: %s', to)
```

Comment:
You should probably also fix the string interpolation. PluginError only takes one argument iirc.